### PR TITLE
[BEAM-6603] Fix Python logging in setup.py

### PR DESCRIPTION
**Pull request by markflyhigh**
Use `warnings` instead of `logging` in `setup.py` since python 3 tests will generate large amount of debug logs by running `python setup.py nosetests`.

+cc: @charlesccychen 

------------------------

Follow this checklist to help us incorporate your contribution quickly and easily:

 - [ ] Format the pull request title like `[BEAM-XXX] Fixes bug in ApproximateQuantiles`, where you replace `BEAM-XXX` with the appropriate JIRA issue, if applicable. This will automatically link the pull request to the issue.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

It will help us expedite review of your Pull Request if you tag someone (e.g. `@username`) to look at it.

Post-Commit Tests Status (on master branch)
------------------------------------------------------------------------------------------------

Lang | SDK | Apex | Dataflow | Flink | Gearpump | Samza | Spark
--- | --- | --- | --- | --- | --- | --- | ---
Go | [Build Status](https://builds.apache.org/job/beam_PostCommit_Go/lastCompletedBuild/) | --- | --- | --- | --- | --- | ---
Java | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Apex/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Dataflow/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Flink/lastCompletedBuild/)<br>[Build Status](https://builds.apache.org/job/beam_PostCommit_Java_PVR_Flink_Batch/lastCompletedBuild/)<br>[Build Status](https://builds.apache.org/job/beam_PostCommit_Java_PVR_Flink_Streaming/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Gearpump/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Samza/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Spark/lastCompletedBuild/)
Python | [Build Status](https://builds.apache.org/job/beam_PostCommit_Python_Verify/lastCompletedBuild/) | --- | [Build Status](https://builds.apache.org/job/beam_PostCommit_Py_VR_Dataflow/lastCompletedBuild/) </br> [Build Status](https://builds.apache.org/job/beam_PostCommit_Py_ValCont/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PreCommit_Python_PVR_Flink_Cron/lastCompletedBuild/) | --- | --- | ---

### CODE REVIEW

**Comment by markflyhigh** [comment]
Log size in Python precommit build is still very large. Disable `--nocapture` doesn't work. Seems those debug logs are generated from python3 tests (both directrunner integration tests and py3-gcp), so the problem may come from Python 3 code.

**Comment by markflyhigh** [comment]
Disable log capture (use `--nologcapture`) reduce the amount of log back to normal. More explanation of this flag in https://nose.readthedocs.io/en/latest/plugins/logcapture.html.

File: sdks/python/tox.ini [hunk 1]
```diff
@@ -61,7 +61,8 @@ commands =
   pip --version
   {toxinidir}/scripts/run_tox_cleanup.sh
   python apache_beam/examples/complete/autocomplete_test.py
-  python setup.py nosetests
+  # Disable log capture due to BEAM-6603.
+  python setup.py nosetests --nologcapture
```
Comment (tvalentyn) [inline sdks/python/tox.ini]: I suggest --logging-level=WARNING here and below

Comment (markflyhigh) [reply sdks/python/tox.ini]: ms

**Comment by markflyhigh** [comment]
PTAL @tvalentyn

**Comment by tvalentyn** [comment]
LGTM, thanks for fixing this!

**Comment by markflyhigh** [comment]
PTAL @tvalentyn

**Comment by tvalentyn** [comment]
Please also change __init__.py. We should use warning there as well instead of warning, since it's semantically more appropriate. Thank you.

**Comment by markflyhigh** [comment]
sg. PTAL @tvalentyn

**Comment by charlesccychen** [comment]
Thanks! This LGTM.

**Comment by markflyhigh** [comment]
retest this please

**Comment by markflyhigh** [comment]
@aaltay Python precommit actually passed but not updated here. PTAL and merge this change to avoid Jenkins Python slowness.

**Pull request merged**